tests/phpunit/payment-methods/class-wc-stripe-express-checkout-element-test.php

```
@@ -78,6 +78,53 @@ public function test_javascript_params() {
+		// Amazon Pay alone keeps the aggregate true regardless of Apple/Google Pay.
+		$helper->method( 'is_amazon_pay_enabled' )->willReturn( true );
+		$helper->method( 'is_link_enabled' )->willReturn( false );
+		$helper->method( 'is_express_checkout_enabled' )->willReturn( true );
+
+		$element = new WC_Stripe_Express_Checkout_Element( $ajax_handler, $helper );
+
+		$actual = $element->javascript_params();
+
+		$this->assertSame( $apple_google_enabled, $actual['stripe']['is_apple_google_pay_enabled'] );
+		$this->assertTrue( $actual['stripe']['is_express_checkout_enabled'] );
+	}
```

PHPMD (2.15.0)

[error] The method test_javascript_params_exposes_method_specific_apple_google_pay_flag is not named in camelCase. (undefined)

(CamelCaseMethodName)